### PR TITLE
Create support-and-triage.md

### PR DESCRIPTION
**Pull request by rianfowler**

### CODE REVIEW

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Oncall support
+
+The oncall developer should handle normal oncall tasks, but may need some frontend support. Helping out or finding the right people to help out can make emergent platform issues easier to deal with.
+
+## Process improvement 
+
+At the end of the rotation, there will be a short handoff meeting where we will discuss
+- ongoing support issues that need to be taken over by the next developer in the rotation
+- patterns in support issues that merit process improvement
```
Comment (cvalarida) [inline teams/vsp/teams/tools/frontend/support-and-triage.md]:
```suggestion
- Ongoing support issues that need to be taken over by the next developer in the rotation
- Patterns in support issues that merit process improvement
```

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Support and Triage rotation 
+
+## Overview 
+
+Supporting VSA teams is requires a cohesive and well coordinated experience. The frontend tools team assigns responsibility for this work to a single person on a weekly rotation. This document summarizes the responsibilities of this developer and how the rotation works. 
+
+## Rotation 
+
+The role will be assigned to a developer for a week, running during our standard support/business hours (Mon - Fri, ~9am - 5pm Eastern). During that time, the responsibilities listed below are that developer's primary responsibility.
```
[the remark this answers is not in the document]
Comment (bkjohnson) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: Same thought here.

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Pull requests 
+
+Ensure that no pull request reviews have been waiting more than 24 hours for a response. This includes follow-ups to reviews. When possible, rely on other FE team members to do PR reviews (e.g. ask for a volunteer and assign the PR to them).
+
+## Support requests
+
+- Create an issue describing the request in GitHub and link it to the Global Customer Support epic (ask Brandon if you need a link to this epic). 
+- Tag the issues with fe-tools, assign it to yourself, and prepend the title of the Ticket with *[Support] -*. 
+- Track any work done with comments. 
+- Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary.
```
Comment (bkjohnson) [inline teams/vsp/teams/tools/frontend/support-and-triage.md]:
```suggestion
- Summarize any solutions and failed attempts so the issue can be referred to again in the future if necessary. 
```

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: updated

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+The rotation schedule can be found [here](https://dsva.pagerduty.com/schedules#PIDMJAN).
+
+## What do I do?
+
+### Triage requests in the support channels
+
+During the sprint, you should monitor the following Slack groups for issues:
+* #vfs-platform-support (individual requests)
+* #vfs-frontend (github reviews)
```
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: updated

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+As currently configured, on the first of the month Dependabot will open PRs until it has 15 active open PRs. These are lower priority, but should be worked through as time permits. [Dependabot filter](https://github.com/department-of-veterans-affairs/vets-api/pulls/app%2Fdependabot-preview)
+
+## Pull requests 
+
+Ensure that no pull request reviews have been waiting more than 24 hours for a response. This includes follow-ups to reviews. When possible, rely on other FE team members to do PR reviews (e.g. ask for a volunteer and assign the PR to them).
+
+## Support requests
+
+- Create an issue describing the request in GitHub and link it to the Global Customer Support epic (ask Brandon if you need a link to this epic).
```
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: I'm on the first rotation so I will get this sorted out and update this document.

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+## Dependabot upgrades 
+
+As currently configured, on the first of the month Dependabot will open PRs until it has 15 active open PRs. These are lower priority, but should be worked through as time permits. [Dependabot filter](https://github.com/department-of-veterans-affairs/vets-api/pulls/app%2Fdependabot-preview)
+
+## Pull requests 
+
+Ensure that no pull request reviews have been waiting more than 24 hours for a response. This includes follow-ups to reviews. When possible, rely on other FE team members to do PR reviews (e.g. ask for a volunteer and assign the PR to them).
+
+## Support requests
```
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: It's every interaction you have with a non-VSP developer.

[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: I think we'll start off tracking everything and then ease up once we have a better view of the picture. also want to make tracking more information easier.

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+# Support and Triage rotation 
+
+## Overview 
+
+Supporting VSA teams requires a cohesive and well-coordinated experience. The frontend tools team assigns responsibility for this work to a VSP front end engineer on a weekly rotation. This document summarizes the responsibilities of this role and how the rotation works.
```
Comment (gunsch) [inline teams/vsp/teams/tools/frontend/support-and-triage.md]: how do we refer to this person? the "support dev"? might be useful to have a role name (to differentiate from "oncall" etc.)

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+If there’s a new error, investigate. If there’s an error that you feel you might be able to fix, start a PR and try. If you need some help, ask for it. The goal is to reduce the number of repeated errors over time so the sentry log is only new, “real” issues.
+
+## Documentation
+
+- Update documentation when resolving support requests. Ideally we have documentation we can link to when resolving support requests. 
+- The backlog has many issues tagged with _documentation_ that can be addressed. 
+
+## Oncall support
+
+The oncall developer should handle normal oncall tasks, but may need some frontend support. Helping out or finding the right people to help out can make emergent platform issues easier to deal with.
```
Comment (gunsch) [inline teams/vsp/teams/tools/frontend/support-and-triage.md]: I'm not sure from reading this paragraph what the expectation here actually is. Is this saying that "support dev should be available for oncall dev to reach out to?" --- or that "support dev should handle FE oncall things?" --- or something else?

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+## Process improvement 
+
+At the end of the rotation, there will be a short handoff meeting where we will discuss
```
Comment (gunsch) [inline teams/vsp/teams/tools/frontend/support-and-triage.md]: between... the whole team? the previous/new support dev?

**Review by gunsch — COMMENTED** [review]
a little late, but here's some thoughts

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: I think we need to adjust this for west coast. for now i changed to to "around" but maybe we should tighten the window to be noon - 1700.

File: teams/vsp/teams/tools/frontend/support-and-triage.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+- Create an issue describing the request in GitHub and link it to the Global Customer Support epic (ask Brandon if you need a link to this epic). 
+- Tag the issues with fe-tools, assign it to yourself, and prepend the title of the Ticket with *[Support] -*. 
+- Track any work done with comments. 
+- Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. 
+- If other team members are involved, add them to the ticket.
+
+## Sentry errors
+
+Browse [website-production Sentry](http://sentry.vfs.va.gov/vets-gov/website-production/) and monitor the #vfs-sentry-alerts channel.
```
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: How do you feel about 

> If time permits, once a day. At least once a rotation. 

Our sentry logs are pretty messy but my hope is that this process will help us put some improvements on our backlog.

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: I'm figuring that out this sprint. I would like to log everything for the first set of rotations but we'll have to see what's actually reasonable. I think we want to push for a helpdesk bot or to build workflows in slack so the more information we have to justify that the better.

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 7 above)
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: support dev works.

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 8 above)
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: > The oncall developer may reach out to the frontend support dev to assist with troubleshooting frontend related issues.

File: teams/vsp/teams/tools/frontend/support-and-triage.md (hunk 9 above)
Comment (rianfowler) [reply teams/vsp/teams/tools/frontend/support-and-triage.md]: >At the end of the rotation, there will be a short handoff meeting between PM, FE lead, and the previous/next rotations' support dev to discuss:

**Pull request merged**